common-controller/internal/operator/operator.go

```diff
@@ -164,14 +170,17 @@ func InitOperator() {
 		loggers.LoggerAPKOperator.ErrorC(logging.PrintError(logging.Error2603, logging.BLOCKER, "Unable to set up ready check: %v", err))
 		os.Exit(1)
 	}
-	config := config.ReadConfigs()
+
 	if config.CommonController.ControlPlane.Enabled {
 		go func() {
 			var controlPlane controlplane.ArtifactDeployer
 			if config.CommonController.ControlPlane.Persistence.Type == "K8s" {
 				controlPlane = controlplane.NewK8sArtifactDeployer(mgr)
 
+			} else if config.CommonController.ControlPlane.Persistence.Type == "DB" {
```

Comment:
Need to add configs to helm charts values.yaml

Reply:
fixed